dactyl-config.yml

```
@@ -4090,20 +4090,13 @@ pages:
             - ja
 
 # Dev Tools --------------------------------------------------------------------
-
-    -   md: dev-tools/dev-tools.md
+    -   name: Dev Tools # Redirect page for old broken URL
```

Looks like you copy-pasted an incorrect comment from the stanza below.

```suggestion
    -   name: Dev Tools
```